src/Tasks/Copy.cs

```diff
@@ -653,12 +667,6 @@ private bool ValidateInputs()
                 return false;
             }
 
-            // First check if create hard or symbolic link option is selected. If both then return an error
-            if (UseHardlinksIfPossible & UseSymboliclinksIfPossible)
-            {
-                Log.LogErrorWithCodeFromResources("Copy.ExactlyOneTypeOfLink", "UseHardlinksIfPossible", "UseSymboliclinksIfPossible");
```

Comment:
You can presumably delete Copy.ExactlyOneTypeOfLink, right?

Reply:
Have deleted that